README.md

```diff
@@ -7,3 +7,59 @@ first line of code to publishing their first project.
 ## Website
 
 Check out the `gh-pages` branch for Hack Camp's website.
+
+## Objective
+
+The objective of Hack Camp is build an set of materials that allows any hackEDU
+team member to reproduce an experience that that brings people from no coding
+knowledge to having the ability, motivation, and confidence to create coding
+projects and continue learning new coding concepts with others, without the
+explicit urging or leading of a teacher.
+
+This can be broken down into two parts: creating the experience and reproducing
+it.
+
+#### Cohorts
+
+##### Cohort 2
+
+The objective of the second cohort is to only do the first part of the Hack Camp
+objective: creating the experience. Our goal is to create an experience that
+brings people from no coding knowledge to having the ability, motivation, and
+confidence to create coding projects and continue learning new coding concepts
+with others, without the explicit urging or leading of a teacher.
+
+##### Cohort 3
+
+The objective of the third cohort is to improve our work from the second cohort.
+We want to do the best job possible satisfying the first part of Hack Camp's
+objective.
+
+##### Cohort 4
+
+The objective of the fourth cohort is to build a set of materials that enables
```

Comment:
Is the objective to build a set of materials? It seems like building these materials could be done in the off week before the cohort. What will we be doing with the materials during the cohort?